apps/system/utils/Kconfig

```diff
@@ -23,7 +23,7 @@ config FS_CMDS
 if FS_CMDS
 config FSCMD_BUFFER_LEN
        int "Buffer length of FSCMD"
-       default 32
+       default 256
```

Comment:
Could you change the defconfigs also?

Reply:
This config is used with local variable, especially `tash_cat`.
How it makes overflow?
`tash_cat` reads and writes based on this value.

Reply:
The old small buffer size causes data to overflow and since there aren't many proper checks in place, the overflow causes a crash.

Reply:
Yes, I know it is from overflow.
What I want to check, we should fix the point which makes overflow.
This is just increasing the buffer size, then it can be overflowed after some time.

Reply:
Yes, this is very much a temporary fix as mentioned.
Proper fixing of multiple ProcFs operations might be required which requires a pre-requisite of checking them as well.
We will be taking that up based on the need of those procfs features and the severity of the issues, just a bit later.

Reply:
In my opinion, this fix is not urgent so that we could make a change with root cause fix.

Reply:
> Could you change the defconfigs also?

Done